File: articles/responsibility-blame-and-moral-accountability/cpp/accountability_score.cpp
/*
Synthetic accountability score example.
Educational scaffold only.

Compile:
  c++ -std=c++17 cpp/accountability_score.cpp -o outputs/tables/accountability_score_cpp
Run:
  ./outputs/tables/accountability_score_cpp
*/

#include <iostream>

int main() {
    double responsibility = 0.8;
    double institutional_structure = 0.6;
    double wrongness_severity = 0.7;
    double repair_capacity = 0.5;
    double prevention_capacity = 0.4;

    double accountability =
        0.45 * responsibility +
        0.25 * institutional_structure +
        0.20 * wrongness_severity +
        0.20 * repair_capacity +
        0.15 * prevention_capacity;

    std::cout << "Synthetic accountability score: " << accountability << std::endl;
    return 0;
}

File: articles/responsibility-blame-and-moral-accountability/go/accountability_score.go
// Synthetic accountability score example.
// Educational scaffold only.
//
// Run:
//   go run go/accountability_score.go

package main

import "fmt"

func main() {
	responsibility := 0.8
	institutionalStructure := 0.6
	wrongnessSeverity := 0.7
	repairCapacity := 0.5
	preventionCapacity := 0.4

	accountability := 0.45*responsibility +
		0.25*institutionalStructure +
		0.20*wrongnessSeverity +
		0.20*repairCapacity +
		0.15*preventionCapacity

	fmt.Printf("Synthetic accountability score: %.3f\n", accountability)
}
